lib/rubocop/rspec/config_formatter.rb

```diff
@@ -19,6 +19,7 @@ def dump
 
       def unified_config
         cops.each_with_object(config.dup) do |cop, unified|
+          next if cop.eql?('RSpec/Cop')
```

Comment:
I found a better way to handle this actually. Good fit for the yard `@abstract` tag